sigmastate/src/main/scala/sigmastate/serialization/OpCodes.scala

```diff
@@ -68,8 +68,7 @@ object OpCodes extends ValueCodes {
   val Undefined: OpCode = OpCode @@ 0.toByte
 
   // variables
-  val TaggedVariableCode: OpCode = newOpCode(1)
-  val ValUseCode: OpCode = newOpCode(2)
+  val ValUseCode: OpCode = newOpCode(2)  // code 1 was used by TaggedVariable (now it is free)
```

Comment:
Please describe how removing an opcode is not a consensus-critical issue. What if a new node will get this opcode?